rcm_nexus/command.py

```diff
@@ -57,79 +83,88 @@ def push(repo, environment, product, version, ga=False, debug=False):
     More information: https://mojo.redhat.com/docs/DOC-1199638
     """
 
-    nexus_config = config.load(environment, debug=debug)
-
-    session = Session(nexus_config, debug=debug)
-    print("Pushing: %s content to: %s" % (repo.decode("utf-8"), environment))
-
-    try:
-        # produce a set of clean repository zips for PUT upload.
-        zips_dir = tempfile.mkdtemp()
-        print("Creating ZIP archives in: %s" % zips_dir)
-        if os.path.isdir(repo):
-            print("Processing repository directory: %s" % repo)
-
-            # Walk the directory tree, and create a zip.
-            zip_paths = archive.create_partitioned_zips_from_dir(repo, zips_dir)
-        else:
-            print("Processing repository zip archive: %s" % repo)
+    nexus_config = config.load(environment)
```

Comment:
We lost the debug mode parameter here...

Reply:
I removed it, because it was not used in any way.